dipy/reconst/ivim.py

```diff
@@ -241,11 +271,11 @@ def __init__(self, gtab, split_b_D=400.0, split_b_S0=200., bounds=None,
         elif self.bounds is None:
             self.bounds = ((0., 0., 0., 0.), (np.inf, .3, 1., 1.))
         else:
-            self.bounds = bounds
+            self.bounds = kwargs.get('bounds', None)
 
     @multi_voxel_fit
-    def fit(self, data, mask=None):
-        """ Fit method of the Ivim model class.
+    def fit(self, data):
```

Comment:
What happened to the mask?

Reply:
Done!

Reply:
The `@multi_voxel` decorator I believe adds the mask and does not need to be included in the fit I guess.. Does that make sense?

-- This is the reason I had taken it down..